### ISSUE TITLE
Incorrect error message for use of type as a namespace

### ISSUE THREAD

**Issue by DanielRosenwasser** [Microsoft/TypeScript#12075]
```ts
interface OhNo {
}

declare let y: OhNo.hello;
//             ~~~~
//             'OhNo' refers to a type, but is being used as a value here.
```

`OhNo` is actually being used as a namespace, not a value. CC @jwbay in case you want to tackle this one. 😄

**Comment by DanielRosenwasser** [comment]
Thanks @mylesmegyesi!
